docs/_data/nakadi-event-bus-api.yaml

```diff
@@ -2251,6 +2251,10 @@ definitions:
             Event identifier of the Event that caused the generation of this Event.
             Set by the producer.
           example: '105a76d8-db49-4144-ace7-e683e8f4ba46'
+      published_by:
```

Comment:
Maybe it's worth mentioning that this field is automatically enriched by nakadi and if provided by the user it will be overwritten.

Reply:
I agree, I added a note for this.